python/ray/tune/execution/checkpoint_manager.py

```diff
@@ -98,12 +99,10 @@ def newest_persistent_checkpoint(self):
         )
 
     @property
-    def newest_checkpoint(self):
-        """Returns the newest checkpoint (based on training iteration)."""
-        newest_checkpoint = max(
-            [self.newest_persistent_checkpoint, self.newest_memory_checkpoint],
-            key=lambda c: c.id,
-        )
+    def checkpoint(self):
```

Comment:
Now that we changed the flag, can we keep the original name? `checkpoint` is very non-descriptive for this base class
```suggestion
    def newest_checkpoint(self):
```

Reply:
Yep, forgot to revert this. `newest_checkpoint` was not exactly correct with the change I had before, which was the reason for the switch.